### PR TITLE
Update parser

### PR DESCRIPTION
**Pull request by WanderingBrooks**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -5,6 +5,9 @@ module.exports = {
     'plugin:prettier/recommended',
     'prettier/react',
   ],
+  parserOptions: {
+    ecmaVersion: 2020,
+  },
```
Comment (joelnedar) [inline index.js:10]: Any background why? @WanderingBrooks

Comment (WanderingBrooks) [reply index.js:10]: I should've put the prs up at the same time but I just sent a message in software dev :)

Comment (WanderingBrooks) [reply index.js:10]: But its because I want to bump the node version

Comment (WanderingBrooks) [reply index.js:10]: It could be that it is repo specific though which parser it should use. So maybe this will be added only to pumbaa

**Pull request closed without merging**